backend/api/v2beta1/job.proto

```diff
@@ -61,10 +61,18 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 
 
 service JobService {
-  // Creates a new job.
-  rpc CreateJob(CreateJobRequest) returns (Job) {
+  // Creates a new job in an experiment, given the experiment ID.
+  rpc CreateJobByExperiment(CreateJobByExperimentRequest) returns (Job) {
     option (google.api.http) = {
-      post: "/apis/v2beta1/jobs"
+      post: "/apis/v2beta1/experiments/{id}/jobs"
```

Comment:
`{id}` needs to comply with a field name in `CreateJobByExperimentRequest`

So, `{id}` needs to become `experiment_id` or via versa